workflows/nallo.nf

```diff
@@ -36,6 +36,7 @@ include { SPLIT_BED_CHUNKS       } from '../modules/local/split_bed_chunks/main'
 include { SAMTOOLS_MERGE         } from '../modules/nf-core/samtools/merge/main'
 
 // nf-core
+include { BCFTOOLS_PLUGINSPLIT   } from '../modules/nf-core/bcftools/pluginsplit/main'
 include { CAT_FASTQ              } from '../modules/nf-core/cat/fastq/'
```

Comment:
Shouldn't it be `'../modules/nf-core/cat/fastq/'main` ?

Reply:
Changed it to main :)